modules/efk-logging-about-curator.adoc

```diff
@@ -5,7 +5,7 @@
 [id="efk-logging-about-curator-{context}"]
 = About Curator in {product-title}
 
-The Elasticsearch Curator tool performs scheduled maintenance operations on a per-index basis. Curator performs actions daily based on its configuration. Only one Curator Pod is
+The Elasticsearch Curator tool performs scheduled maintenance operations on a global and/or on a per-project basis. Curator performs actions daily based on its configuration. Only one Curator Pod is
 recommended per Elasticsearch cluster. 
```

Comment:
remove the `Only one Curator Pod is recommended per Elasticsearch cluster.` since it is a CronJob now.

Reply:
Thank you . The sentence was in there twice and I removed only one. Fixed!